gpu_bdb/benchmark_runner.py

```diff
@@ -89,8 +90,17 @@ def load_query(qnum, fn):
             with open("current_query_num.txt", "w") as fp:
                 fp.write(qnum)
 
+            rmm_analyzer=RMMResourceMonitor(client=client,
```

Comment:
Can we make this optional? Possibly by an env/yaml switch in cluster_startup.sh?

Reply:
Definitely, that is one of 2 TBD items in the PR summary.